JitsiConference.js

```diff
@@ -367,11 +367,17 @@ JitsiConference.resourceCreator = function(jid, isAuthenticatedUser) {
  * @param options {object}
  * @param options.connection {JitsiConnection} overrides this.connection
  */
+
+  /**
+   * Class JitsiConferenceOptions:
```

Comment:
This should be blended with the above comment since it describes the class already.

Reply:
I have made the changes you suggested